File: javascript/isogram/isogram.js
function Isogram(word) {
  var chars = word.toLowerCase().replace(/\s|-/g, "").split("");
  this.isIsogram = function() {
    return chars.filter((e, _i, self) => self.indexOf(e) === self.lastIndexOf(e)).length === chars.length;
  };
};

module.exports = Isogram;

File: csharp/isogram/Isogram.cs
using System.Linq;

class Isogram
{
	public static bool IsIsogram(string input)
	{
		string lowered = input.ToLower();
		return input.Length == lowered.Where(x => lowered.IndexOf(x) == lowered.LastIndexOf(x) || (x == ' ' || x == '-')).Count();
	}
}
